docs/building.md

```diff
@@ -2,22 +2,39 @@
 
 You must have a 64-bit machine for building and running the project. Always
 run your system updater before building and make sure you have the latest
-video drivers for your card.
+drivers.
 
 ## Setup
 
 ### Windows
 
-* Windows 8 or later
-* [Visual Studio 2017 or Visual Studio 2019](https://www.visualstudio.com/downloads/)
-* [Python 3.4+](https://www.python.org/downloads/)
-* You may also need the [Windows 8.1 SDK](https://msdn.microsoft.com/en-us/windows/desktop/bg162891)
-  * (for VS2017 just click the Windows 8.1 SDK option in the Individual Components section in the Visual Studio Installer)
+* Windows 7 or later
+* [Visual Studio 2019 or 2017](https://www.visualstudio.com/downloads/)
+* [Python 3.4+](https://www.python.org/downloads/) installed and on PATH.
```

Comment:
Would be better to make the PATH note a subitem. IE:

* [Python 3.4+](https://www.python.org/downloads/)
    * Ensure Python is in PATH.